pyproject.toml

```diff
@@ -101,7 +101,7 @@ omit = [
 
 [tool.coverage.report]
 precision = 2
-fail_under = 98.35
+fail_under = 98.13
```

Comment:
I don't want to decrease this, can we add pragmas where needed?

Reply:
@abersheeran can you check what's missing here? I'm already fine with the implementation.

Reply:
> @abersheeran can you check what's missing here? I'm already fine with the implementation.

I will solve it tonight. I'm glad to hear that you like this implementation.